src/package/abi/Microsoft.Windows.AbiWinRT.IdlGen.targets

```diff
@@ -0,0 +1,83 @@
+<!--
+***********************************************************************************************
+Copyright (C) Microsoft Corporation. All rights reserved.
+***********************************************************************************************
+-->
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <!--
+        This file contains targets for generating IDL files from winmd references. This are only
+        used when the consumer hasn't switched to Modern IDL.
+    -->
+    
+  <!--If not using the newer modern idl semantics, generate the .idl for any referenced winmd files -->
+  <Target Name="GetAbiWinRTDirectWinMDReferencesForIdl"
+          BeforeTargets="Midl"
+          Returns="@(AbiWinRTDirectWinMDReferencesForIdl);@(AbiWinRTDirectWinMDReferenceDirs)">
+    <ItemGroup>
+      <_AbiWinRTDirectWinMDReferencesForIdl Remove="@(_AbiWinRTDirectWinMDReferencesForIdl)" />
+      <_AbiWinRTDirectWinMDReferencesForIdl Include="@(Reference)" Condition="'%(Reference.Extension)' == '.winmd' and '%(Reference.IsSystemReference)' != 'true'" />
+      <AbiWinRTDirectWinMDReferencesForIdl Remove="@(AbiWinRTDirectWinMDReferencesForIdl)"/>
+      <AbiWinRTDirectWinMDReferencesForIdl Include="@(_AbiWinRTDirectWinMDReferencesForIdl)">
+        <WinMDPath>%(FullPath)</WinMDPath>
+      </AbiWinRTDirectWinMDReferencesForIdl>
+
+      <!-- Generate list of reference directories to pass into winmdidl.exe -->
```

Comment:
tool name is abi.exe

Reply:
this was for running winmidl, but is no longer part of this change